src/source-updater.js

```diff
@@ -41,16 +71,66 @@ export default class SourceUpdater {
       this.runCallback_();
     };
 
+    // Fixes https://github.com/videojs/videojs-contrib-hls/issues/963
+    // We run this on the next tick as it seems
+    // that sometimes when the `sourceopen` event is triggered, or when
+    // the readyState indicates open, the
+    // MediaSource is actually not ready yet to accept
+    // calls to its `addSourceBuffer` method.
+    // This seems to happen especially for when we need to create SourceBuffers
+    // for seperate audio streams (A/V not muxed into the "main" stream).
+    // However when state is settled on the next tick,
+    // it seems safe to do so.
+    createSourceBufferDeferred =
+      (() => setTimeout(createSourceBuffer, ADD_SOURCE_BUFFER_RETRY_DEFER_MS));
```

Comment:
running this immediately (zero timeout) doesn't fix the issue. 

we have experienced that this type of "delay" allows for a reliable behavior.

that proves this is a pretty "fuzzy" issue in the lower layers imho.